changelog-updater.js

```diff
@@ -8,7 +8,11 @@ const version = await $`bunx changelog --latest-release`.text();
 const packageVersion = await $`npm info vite-plugin-fable version`.text();
 
 if (version === packageVersion) {
-  process.exit(0);
+  console.log("version is already up to date, do not publish");
+  process.exit(1);
```

Comment:
if same version, return error / false - no need to publish or further actions in publication